core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -503,6 +506,10 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
       dataSource.inTransaction { dataAccess =>
         withWorkspaceContext(workspaceName, dataAccess) { workspaceContext =>
           val dedupedInvites = invites.filterNot(update => existingInvites.contains(WorkspaceACLUpdate(update.email, update.accessLevel, None)))
+
+          val invitedNotifications = dedupedInvites.map(invite => Notifications.WorkspaceInvitedNotification(invite.email, userInfo.userEmail))
```

Comment:
i think we need to do another round of filtering here before we actually get to the list of invite emails that we want to send. basically, if someone updates the `accessLevel` of a pending invite, we don't want to send them another invite to FireCloud

`dedupedInvites` filters out any invites that are identical to ones that already exist, but we still need to do something like: https://github.com/broadinstitute/rawls/pull/638/files#diff-91fe8efb10dfe91234a26e4f80039fc6R525